lib/flutter/flutter_launch.dart

```diff
@@ -30,7 +30,9 @@ class FlutterLaunchType extends LaunchType {
 
   String get flutterRunCommand => 'run';
 
-  bool canLaunch(String path) {
+  bool canLaunch(String path, { LaunchData data }) {
+    if (data == null) return false;
```

Comment:
This was mostly an artifact of the refactoring to get here; not all clients use it but I can make it a normal param in any case.